### ISSUE TITLE
Need a way to ask about the question

### ISSUE THREAD

**Issue by vvorski** [fairdemocracy/vilfredo-client#220]
There is a place to submit answers, and a way to ask questions about an answer, but no way to ask about the question and discuss it.

Perhaps at least need a link to the appropriate hangout, or slack channel, or skype group, etc. where the question is being discussed?

**Comment by vvorski** [comment]
@pietrosperoni  @giovanisp  what do you think?

**Comment by pietrosperoni** [comment]
yes and no. The risk is that people use that to decide the answer and shortcut the result making the people who follow vilfredo feel cut out. 
So formal questions and request for clarifications yes. 
("request for clarification" @giovanisp, what about it also for the proposal)
but a chat channel absolutely not (nice try, btw)

**Comment by pietrosperoni** [comment]
@giovanisp , not sure how to address this. I would change this into open a Vilfredo question on how to discuss a question, before posting the proposals. So nothing here yet. And even there it is important thatthe discussion is only about the way the question is formulated, not the actual proposals. Or it breaks vilfredo

**Comment by pietrosperoni** [comment]
maybe the best thing is to just let the question text be a wiki for some time.

**Comment by vvorski** [comment]
People will just chat in google hangouts, Skype, Facebook and at the watercooler, in the sauna and at the gym.

Not even the Chinese government can stop people talking to each other...

V/.

**Comment by vvorski** [comment]
Changing the question is separate from discussing it. 

Google wave tried to mix discussion and the text being edited... We all know where google wave ended up....

from that I learn that discussion about something and working on it must be separate layers.

V/.

**Comment by pietrosperoni** [comment]
it's different. A Vilfredo question has a selected audience. Every time exactly those people meet they have the potentiality to decide there and then. But then they will not do it fairly. We cannot stop them from meeting, but we can at least not give them the opportunity while the question is running for them to do it. If then two friends meet up for a beer, no big deal. But Vilfredo gives the possibility to take decisions online. And people need to schedule those meetings in general. We are not making this easier, sorry. 

Please @vvorski  try to understand Vilfredo before trying to change it.

**Comment by giovanisp** [comment]
@pietrosperoni  we need simpler and clearer explanations of "what is Vilfredo" and "what it tries to do". Take a look at the Wiki in vilfredo-core. Maybe @vvorski don't fully understand because it is not written clearly anywhere? Also, if a person inside our group like him has problems undestanding the underlying principles of Vilfredo, we need to think well about why that happens and get their help on what we are doing wrong.

@vvorski , yes, people can (and will) talk about proposals using different channels. And they should do that. What Vilfredo will do however is not encourage public conversations that can "shirt circuit" the decision-making process. Decisions will follow a given path and Vilfredo will suggest the better people you should talk to, to achieve a fair result and spread participation (that's why processes exist). Adding such features now is premature, since we did not nailed the decision making process yet (think the multiple generations, clarifications and so on...). Doing this now can turn Vilfredo into "an app like all others" for decision-making. It is not. At least it tries to bring a higher quality result respect other apps out there. So if you agree, we could keep this on hold until we reach v2.4  (see the milestones roadmap). Also, keep in mind that we are running on extremely low resources for coding, so we are prioritizing the core functionality and algorithms research, because we cannot do otherwise :-)  Please keep posting your experiences, it is great to have people like you pioneering the use of such a system.

**Comment by pietrosperoni** [comment]
Thanks @giovanisp , I will work on the tutorial. RIght now I am writing the subtitles in the italian tutorial into English. The reason why I am doing this is because that was the best explanation I have ever given.

**Comment by pietrosperoni** [comment]
We should put some issues about the tutorial for me to do and spread them on the timeline. it would be great for me to know what to do and when.
